scripts/download/wsl.mjs

```diff
@@ -9,7 +9,7 @@ import { download } from '../lib/download.mjs';
 
 export default async function main() {
   await download(
-    'https://github.com/jandubois/tinyk3s/releases/download/v0.1/distro.tar',
-    path.resolve(process.cwd(), 'resources', os.platform(), 'distro-0.1.tar'),
+    'https://github.com/jandubois/tinyk3s/releases/download/v0.2/distro-0.2.tar',
```

Comment:
Please update repo from `jandubois/tinyk3s` to `rancher-sandbox/rancher-desktop-wsl-distro`

Reply:
Thanks! Fixed in 7d68d09ce3aa2991f32d7922f7d63ff5923119a2.
(Also, rebased because I forgot to run the linter…)